ppafm/data.py

```diff
@@ -7,9 +7,10 @@
 from urllib.request import urlretrieve
 
 DATASET_URLS = {
-    "CO-tip-densities": "https://zenodo.org/records/10418629/files/CO_tip_densities.tar.gz?download=1",
-    "dft-afm": "https://zenodo.org/records/10418629/files/dft-afm.tar.gz?download=1",
-    "hartree-density": "https://zenodo.org/records/10418629/files/hartree-density.tar.gz?download=1",
+    "CO-tip-densities": "https://zenodo.org/records/10563098/files/CO_tip_densities.tar.gz?download=1",
+    "dft-afm": "https://zenodo.org/records/10563098/files/dft-afm.tar.gz?download=1",
+    "hartree-density": "https://zenodo.org/records/10563098/files/hartree-density.tar.gz?download=1",
+    "FFPB_KPFM_hartree": "https://zenodo.org/records/10563098/files/KPFM_hartree.tar.gz?download=1",
```

Comment:
In order to follow the same scheme, I would change the underscores `_`, to dashes `-` in the name.

Reply:
Okay, changed.